src/main/java/productDependency/CreateDependentCategoryPage.java

```diff
@@ -0,0 +1,974 @@
+		log.info("Enter Price Rate");
+		JavaScriptExec.sleep();
+		WaitClass.WaitForElementisDisplay(driver, 10, enterPriceRate);
+		Assert.assertTrue(enterPriceRate.isDisplayed());
+		//enterPriceRate1.clear();
+		enterPriceRate.sendKeys(BasePage.getCellData(xlsxName, sheetName, 6, 3));
+
+}
+	@FindBy(how = How.XPATH, using = "//a[@class='submit add']//*[text()='Add Price']")
+	private WebElement clickAddPriceButton;
+
+	/**
+	 * Method to click on Add Price Button.
+	 * 
+	 * @throws IOException
+	 */
+	public void clickAddPriceButton() throws IOException {
+		CreateDependentCategoryPage sp = new CreateDependentCategoryPage(driver);
+		log.info("Click on Add Price Button");
+		Assert.assertTrue(clickAddPriceButton.isDisplayed());
+		clickAddPriceButton.click();
+		//navigateBottom();
+
+	}
+	
+	@FindBy(how = How.XPATH, using = "//*[@id='dependency']/div[1]/a/span")
+	private WebElement clickDependenciesPlus;
+
+	/**
+	 * Method to click on Dependencies plus icon.
+	 * 
+	 * @throws IOException
+	 */
+	public void clickDependenciesPlus() throws IOException {
+		CreateDependentCategoryPage sp = new CreateDependentCategoryPage(driver);
+		log.info("click on Dependencies plus icon");
+		Assert.assertTrue(clickDependenciesPlus.isDisplayed());
+		clickDependenciesPlus.click();
+		JavaScriptExec.sleep();
+	}
+
+	private WebElement selectProductCategoryDependency;
+
+	/**
+	 * Method to select product category in dependencies section.
+	 * 
+	 * @throws IOException
+	 */
+	public void selectProductCategoryDependency() throws IOException {
+		CreateDependentCategoryPage sp = new CreateDependentCategoryPage(driver);
+		WebElement PCelement = driver.findElement(By.xpath("//select[@name='product.dependencyItemTypes']"));
+		Select se = new Select(PCelement);
+		se.selectByVisibleText(BasePage.getCellData(xlsxName, sheetName, 3, 0));
+
+	}
+
+	private WebElement selectProductDependency1;
+
+	/**
+	 * Method to select product in dependencies section.
+	 * 
+	 * @throws IOException
+	 */
+	public void selectProductDependency1() throws IOException {
+		CreateDependentCategoryPage sp = new CreateDependentCategoryPage(driver);
+		JavaScriptExec.sleep();
+		driver.findElement(By.xpath("//*[@id='product.dependencyItems']/option[4]")).click();
+		//*[@id="product.dependencyItems"]/option[3]
+		
+		
+		
+
+	}
+
+	private WebElement selectProductPeriod;
+
+	/**
+	 * Method to select product period in dependencies section.
+	 * 
+	 * @throws IOException
+	 */
+	public void selectProductPeriod() throws IOException {
+		CreateDependentCategoryPage sp = new CreateDependentCategoryPage(driver);
+		WebElement Productelement = driver.findElement(By.xpath("//select[@name='period']"));
+		Select se = new Select(Productelement);
+		se.selectByVisibleText(BasePage.getCellData(xlsxName, sheetName, 7, 0));
+
+	}
+	
+	@FindBy(how = How.XPATH, using = "//input[@id='product.dependencyMin']")
+	
+	//*[@id="product.dependencyMin"]
+	private WebElement enterMin;
```

Comment:
@mayankchhajer remove this commented xpath and remove the lines between xpath and WebElement

Reply:
Done